build-bin/docker-compose.test.yml

```diff
@@ -2,13 +2,45 @@
 version: "2.4"
 
 services:
+  sut:
```

Comment:
@anuraaga @jcchavezs this is a sneaky way to put a blocking loop on a trace being found